ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts

```diff
@@ -10,7 +10,9 @@ export type CustomNodeModel =
     | DeploymentNodeModel
     | ExternalNodeModel
     | ExternalEntitiesNodeModel
-    | CIDRBlockNodeModel;
+    | CIDRBlockNodeModel
+    | PolicyNodeModel
```

Comment:
What is the `PolicyNodeModel` and how is it different from a `DeploymentNodeModel`? I was wondering more why have a separate type. If a `PolicyNodeModel` has `nonIsolatedIngress` and `nonIsolatedEgress` as false, then would it be the same as a `DeploymentNodeModel`?